src/Hogwarts.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Sorting from './images/sortinghat.jpg';
+import './Hogwarts.css';
+import './App';
+export default function Hogwarts(props){ 
+    return(
+        <section className="cuadroHogwarts">
+            <article className="Contenedorimagen">
+                <img src= {Sorting} id="imagen50"></img>
+            </article>
+            <article className="ContenedorTexto">
+                <h1 id="titulo" >The Sorting Hat</h1>
+                <p id="parrafo" >The Sorting Hat is one of the cleverest enchanted objects most witches and wizards will ever meet. It literally contains the intelligence of the four founders, can speak (through a rip near its brim) and is skilled at Legilimency, which enables it to look into the wearer’s head and divine his or her capabilities or mood.</p>
+                <p id="autor" >J.K. Rowling</p>
+                <button type="button" id="Boton">READ MORE</button>
```

Comment:
No es incorrecto agregar estilos a los elementos con ids, pero el uso principal de los ids es para diferenciar un elemento único. En este caso cada elemento era único porque no hay más elementos en la app 😅, pero siempre intenta pensarlo en una escala más grande. Si esto fuera una aplicación más grande probablemente vas a querer re-utilizar las reglas de estilos de los botones, de los títulos, del contenido, imagen, etc. Entonces rule-of-thumb "Siempre usar clases, si el elemento es súper único e irrepetible, no hay nadie cómo él, es Michael Jackson, es la última coca del desierto, entonces usar id"